pkg/gcc/interceptor.go

```diff
@@ -195,7 +194,7 @@ func (c *Interceptor) BindLocalStream(info *interceptor.StreamInfo, writer inter
 
 	c.pacer.AddStream(info.SSRC, interceptor.RTPWriterFunc(func(header *rtp.Header, payload []byte, attributes interceptor.Attributes) (int, error) {
 		// Call adapter.onSent
-		if err := c.OnSent(time.Now(), header, len(payload), attributes); err != nil {
+		if err := c.OnSent(time.Now(), header, len(payload), attributes); err != nil && err != errMissingTWCCExtension {
```

Comment:
I have a question about this: Does it make sense to send the stream through this interceptor at all? Could we just do:
```
	var hdrExtID uint8
	for _, e := range info.RTPHeaderExtensions {
		if e.URI == transportCCURI {
			hdrExtID = uint8(e.ID)
			break
		}
	}
	if hdrExtID == 0 {
		return writer
	}

	c.pacer.AddStream(info.SSRC, interceptor.RTPWriterFunc(func(header *rtp.Header, payload []byte, attributes interceptor.Attributes) (int, error) {
```
instead, to bypass the whole interceptor? (header extension IDs can't be 0).